### DISCUSSION TITLE
2023 홈페이지 개편 이슈사항

### DISCUSSION THREAD

**Discussion by shanmdphd** [pipetcpt/introduction#19] [Ideas]
아래에 코멘트 남겨주세요.

기한 : 2023-07-28 (금) 18:00

**Comment by shanmdphd** [comment]
- [ ] Research > Pharmacotranscriptomics 그림 좀더 fancy한거로 바꿔보자. 
- [x] Admin page 접속 오류

**Comment by junghyun0** [comment]
- [x] organization: faculty 부분 수정 필요 
<img width="1037" alt="스크린샷 2023-07-26 오후 1 01 34" src="https://github.com/pipetcpt/introduction/assets/126430221/f668e193-2aea-4108-b7f4-e3eaa042aa4c">

**Reply by shanmdphd** [reply]
GOOD

**Comment by shanmdphd** [comment]
- [x] Notice 에서 공지사항 입력시 Text Editor 동작 안함. 줄바꿈 안됨. http://pipet.or.kr/board/notice_view.asp?curPage=1&search=&keyword=&idx=1061
